logic.py

```diff
@@ -228,6 +228,10 @@ def get_setting(name, journal):
     return setting_handler.get_setting('plugin:ezid', name, journal).processed_value
 
 def get_journal_metadata(article):
+    # get remote_url from articles
+    itemId = 'qt'+ article.remote_url[-8:]
```

Comment:
Should the comment reflect that `remote_url[-8:]` gets the ark, and not the whole remote url?

Reply:
Yes, need updated comment and add a check remote_url before accessing it.